src/ansys/mapdl/core/_commands/aux2_/bin_manip.py

```diff
@@ -21,323 +21,315 @@
-        damping matrix values) are incorporated into the globally assembled
-        stiffness matrix.  Thus, the globally assembled stiffness matrix
-        represents more than the stiffness of the model for these analysis
-        types.  Please refer to the Mechanical APDL Theory Reference for more
-        details.
-
-        When dumping a .FULL file, the rows and columns corresponding to
-        specified constraints (e.g., D commands) are eliminated from the system
-        of equations and therefore are not written to the .MATRIX file. Also,
-        rows and columns corresponding to eliminated (slave) degrees of freedom
-        from coupling and/or constraint equations (e.g., CE, CP commands) are
-        also eliminated from the system of equations and are not written to the
-        .MATRIX file. The DOFs that are eliminated from any coupling and/or
-        constraint equations are determined internally by the solution code and
-        may not match what you specified via the CE/CP (or similar) commands.
-
-        When dumping a .SUB file, the full nxn matrix will be written to the
-        .MATRIX file for either symmetric or unsymmetric matrices, regardless
-        of whether any of the matrix coefficients are zero-valued. When dumping
-        a .FULL file, only the lower triangular part of the matrix will be
-        written to the .MATRIX file if the matrix is symmetric; the full matrix
-        is written if the matrix is unsymmetric. Only matrix coefficients that
-        are greater than zero will be written.
-
-        The Harwell-Boeing format is column-oriented.  That is, non-zero matrix
-        values are stored with their corresponding row indices in a sequence of
-        columns.  However, since the ANSYS matrix files are stored by row and
-        not column, when the HBMAT command is used with a non-symmetric matrix,
-        the transpose of the matrix is, in fact, written.
-
-        The WRFULL command, in conjunction with the SOLVE command, can be used
-        to generate the assembled global matrix file and eliminate the equation
-        solution process and results output process.
-
-        The mapping file can be used to map the matrix equation numbers found
-        in the .MATRIX file directly to the corresponding node numbers and
-        degrees of freedom.
-
-        When dumping a CMS .SUB file, the last rows/columns of the matrix are
-        non-physical degrees of freedom added internally by the CMS process and
-        cannot be mapped directly to a node number or particular degree of
-        freedom.
+         DMP solution that generated the files.
+
+        When running on a cluster, the local solution files must be available in the working directory on
+        each node in the subsequent session. As an example, consider the following command line used to
+        generate local solution files:
+
+        .. code:: apdl
```

Comment:
This is ``..code:: console``.

Reply:
Making a difference between ``console`` code and ``apdl`` one is complex.
Which one do you prefer as the default one?
Differentiation can be added for further development